lib/Runtime/Library/JavascriptDate.cpp

```diff
@@ -1503,16 +1503,6 @@ namespace Js
             DateImplementation::DateTimeFlag::NoDate);
     }
 
-    // CONSIDER: ToGMTString and ToUTCString is the same, but currently the profiler use the entry point address to identify
-    // the entry point. So we will have to make the function different. Consider using FunctionInfo to identify the function
```

Comment:
So until this morning, I thought this profiling was like the observable information from F12, and that it wouldn't be a big deal to just treat this as the same function for that purpose. Even if that's not true, since it is literally the same function now aside from the name, I think it might still be okay to treat these as the same function. We can discuss offline if this is a deeper issue than I realize.